### PR TITLE
ui#915 - set limits by user category

### PR DESCRIPTION
**Pull request by qijieye**

### CODE REVIEW

File: Client/Categories/index.ts [hunk 1]
```diff
@@ -14,4 +14,13 @@ export class Categories {
 		else
 			return result
 	}
+	async setCategoryLimits(
+		category: string,
+		request: model.CategoryLimitRequest
+	): Promise<model.CategoryResponse | model.ErrorResponse> {
+		return await this.connection.put<model.CategoryResponse>(`category/${category}`, request)
+	}
```
Comment (lucy-meachem) [inline Client/Categories/index.ts]: should be called "editCategory" or something

File: Client/Users/index.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class Users extends Collection<model.UserResponse, model.UserSearchReques
 		)
 		return this.extractResponse(response)
 	}
-	async getCategory(): Promise<string[] | model.ErrorResponse> {
+	async getUserCategory(): Promise<string[] | model.ErrorResponse> {
 		const result = await this.connection.get<string[]>(`users/category`)
 		return result
 	}
```
Comment (lucy-meachem) [inline Client/Users/index.ts]: please don't use this method - use `GET /category` to do the same thing (get the categories)

File: model/CategoryLimitRequest.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { CategoryFundingAccountAccessRequest } from "./CategoryFundingAccountAccessRequest"
+import { UserLimitsRequest } from "./UserLimitsRequest"
+
+export interface CategoryLimitRequest {
```
Comment (lucy-meachem) [inline model/CategoryLimitRequest.ts]: UpdateCategoryRequest would make more sense

File: model/CategoryLimitRequest.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+	limits?: UserLimitsRequest[]
```
Comment (lucy-meachem) [inline model/CategoryLimitRequest.ts]: ive changed the object to accept setLimits, use that instead? (the old one still works too)

File: Client/Users/index.ts (hunk 2 above)
Comment (qijieye) [reply Client/Users/index.ts]: we have this in category folder so just gonna delete this one from users (already checked no place is using this)

**Review by lucy-meachem — APPROVED** [review]

**Pull request merged**